docs/getting-started/unreal-editor/index.md

```diff
@@ -50,7 +50,21 @@ should be copied to `{PackagingStagingFolder}/PAYDAY3/Maps/CoolMap/CoolMap.umap`
 Once all your files are copied, you must now package the `PackageStagingFolder` with the tool of your choice.
 The tool you use is up to personal preference, but the recommended tool is [repak by trumank](https://github.com/trumank/repak)
 
-Once packaged, place your pak files in `{InstallFolder}/PAYDAY3/Content/Paks/` and your mod should be loaded by PAYDAY 3.
+1. Add [repak](https://github.com/trumank/repak/releases/latest) to your [`PATH`](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53) or drop the exe next to the folder you want to pak (`PackageStagingFolder` in this case)
+2. Open a `cmd` or `powershell` window in the folder by holding the `Shift` key and right clicking on the explorer background, then select in the context menu select "Open [command or powershell] window here".
+3. Run `repak.exe pack PackageStagingFolder` to pack `PackageStagingFolder` into `PackageStagingFolder.pak`.
```

Comment:
We should probably recommend `PackageStagingFolder_P`

Reply:
Perhaps `MyNewMod_P` to indicate it's just the mod name then?